### PR TITLE
Adding a new tool called CakeFuzzer

### PR DESCRIPTION
**Pull request by denizilhan**
I added our interactive application security testing tool called CakeFuzzer. CakeFuzzer employs both static and dynamic testing techniques to find all possible endpoints of the application and conduct security tests.

* [x] I have not changed the `README.md` directly.

### CODE REVIEW

File: data/tools/cakefuzzer.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  - linter
+tags:
+  - php
+license: GNU GPL
+types:
+  - cli
+source: 'https://github.com/Zigrin-Security/CakeFuzzer'
+homepage: 'https://zigrin.com/tools/cake-fuzzer/'
+description: >-
+  The ultimate web application security testing tool for CakePHP-based web applications. CakeFuzzer employs a predefined set of attacks that are randomly modified before execution. Leveraging its deep understanding of the Cake PHP framework, Cake Fuzzer launches attacks on all potential application entry points.
```
[the remark this answers is not in the document]
Comment (mre) [reply data/tools/cakefuzzer.yml:12]: We aim for a neutral tone for all tools on the list to avoid bias.

**Comment by mre** [comment]
Hi,

is this, strictly speaking, a static analysis tool?
The `README.md` mentions Interactive Application Security Testing (IAST) as a sort of hybrid static/dynamic approach.

**Comment by denizilhan** [comment]
Hello,

Yes, CakeFuzzer is an IAST tool that employs SAST techniques, and source code analysis and combines it with a dynamic approach to minimize false positives. I also mentioned in the PR. Please ask any further questions since it is truly a unique tool and generated really good real-world results.

**Comment by mre** [comment]
All good. I took a closer look, and it makes sense to add it. If you're fine with accepting the suggestion I made, I'm happy to merge it. 😃

**Comment by denizilhan** [comment]
Yes, I am fine and understand the reason :) thank you

**Comment by mre** [comment]
Awesome. Thanks a lot for the contribution. 😃

**Pull request merged**